nni/algorithms/compression/v2/pytorch/pruning/basic_pruner.py

```diff
@@ -48,27 +48,31 @@
 NORMAL_SCHEMA = {
     Or('sparsity', 'sparsity_per_layer'): And(float, lambda n: 0 <= n < 1),
     SchemaOptional('op_types'): [str],
-    SchemaOptional('op_names'): [str]
+    SchemaOptional('op_names'): [str],
+    SchemaOptional('op_partial_names'): [str]
 }
 
 GLOBAL_SCHEMA = {
     'total_sparsity': And(float, lambda n: 0 <= n < 1),
     SchemaOptional('max_sparsity_per_layer'): And(float, lambda n: 0 < n <= 1),
     SchemaOptional('op_types'): [str],
-    SchemaOptional('op_names'): [str]
+    SchemaOptional('op_names'): [str],
+    SchemaOptional('op_partial_names'): [str]
 }
 
 EXCLUDE_SCHEMA = {
     'exclude': bool,
     SchemaOptional('op_types'): [str],
-    SchemaOptional('op_names'): [str]
+    SchemaOptional('op_names'): [str],
+    SchemaOptional('op_partial_names'): [str]
 }
 
 INTERNAL_SCHEMA = {
     'total_sparsity': And(float, lambda n: 0 <= n < 1),
     SchemaOptional('max_sparsity_per_layer'): {str: float},
     SchemaOptional('op_types'): [str],
-    SchemaOptional('op_names'): [str]
+    SchemaOptional('op_names'): [str],
+    SchemaOptional('op_partial_names'): [str]
```

Comment:
`op_partial_names` should not in `INTERNAL_SCHEMA`

Reply:
OK, I got it. And next I will add something in need in 'config_validation.py' as well.